### PR TITLE
Integrate gutenberg-mobile release v1.112.0

### PR DESCRIPTION
**Pull request by derekblank**
## Description
This PR incorporates the 1.112.0 release of gutenberg-mobile.
For more information about this release and testing instructions, please see the related Gutenberg-Mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/6597

Release Submission Checklist

- [ ] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">PR is not assigned to a milestone.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by fluiddot — COMMENTED** [review]
Following the Release Submission Checklist in the PR:
> I have considered if this change warrants user-facing release notes and have added them to RELEASE-NOTES.txt if necessary.

It would be interesting to check if this release introduced user-facing changes (e.g. _Prevent images from temporarily disappearing when uploading media_) and if so, reflect it.

**Review by fluiddot — APPROVED** [review]
LGTM 🎊 !

**NOTE:** Before merging this PR, we need to update the Gutenberg Mobile reference to point to the release tag. Besides, we could review the release notes as mentioned in https://github.com/wordpress-mobile/WordPress-Android/pull/20093#pullrequestreview-1856374475.

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is assigned to the milestone <a href="https://github.com/wordpress-mobile/WordPress-Android/milestone/276">24.2</a>. This milestone is due in less than 4 days.<br />
Please make sure to get it merged by then or assign it to a milestone with a later deadline.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -4,6 +4,7 @@
 -----
 * [**] Fix editor crash occurring on large posts [https://github.com/wordpress-mobile/WordPress-Android/pull/20046]
 * [*] [Jetpack-only] Site Monitoring: Add Metrics, PHP Logs, and Web Server Logs under Site Monitoring [https://github.com/wordpress-mobile/WordPress-Android/issues/20067]
+* [**] Prevent images from temporarily disappearing when uploading media [https://github.com/WordPress/gutenberg/pull/57869]
```
Comment (fluiddot) [inline RELEASE-NOTES.txt:7]: @derekblank We usually put the prefix `Block Editor: ` in the release notes for entries related to the editor ([example](https://github.com/wordpress-mobile/WordPress-Android/blob/9aaa1d4410fd2094ae8f1b82128f0a8ac28e603b/RELEASE-NOTES.txt#L22)).

**Pull request merged**